test/test_bar.py

```diff
@@ -133,7 +137,7 @@ def test_draw(manager):
 
 
 @gb_config
-def test_prompt(manager):
+def test_prompt(manager, monkeypatch):
```

Comment:
Why do we need `monkeypatch` here? It doesn't seem to be in your test.

Reply:
Ah yes, good spot. I added it for a first draft as I was changing $PATH but then realised i didn't need to. Will remove!